app/views/chefs/_form.html.erb

```diff
@@ -3,9 +3,9 @@
 <div class="row">
   <%= form_for(@chef, :html => {role: "form"}) do |f| %>
     <div class="col mb-3">
-      <%= f.label :chefname, "Choose Chefname", class: "form-label" %>
+      <%= f.label :name, "Choose name", class: "form-label" %>
```

Comment:
Changing the label was not in the AC but I think it makes sense. Hopefully the PO approves it!